components/nexus-repository/src/main/java/org/sonatype/nexus/repository/config/ConfigurationFacetImpl.java

```diff
@@ -42,6 +44,9 @@
     extends FacetSupport
     implements ConfigurationFacet
 {
+  private static final TypeReference<Map<String, Object>> MAP_STRING_OBJECT =
+      new TypeReference<Map<String, Object>>() {};
```

Comment:
I may turn this rule off, its not smart enough presently to know that empty { } can/should live on the same line.

—jason




On July 17, 2015 at 7:28:34 AM, Sonatype Zion (notifications@github.com) wrote:

In components/nexus-repository/src/main/java/org/sonatype/nexus/repository/config/ConfigurationFacetImpl.java:

> @@ -42,6 +44,9 @@
>      extends FacetSupport
>      implements ConfigurationFacet
>  {
> +  private static final TypeReference<Map<String, Object>> MAP_STRING_OBJECT =
> +      new TypeReference<Map<String, Object>>() {};
 Move this closing curly brace to the next line.   

—
Reply to this email directly or view it on GitHub.